oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -560,7 +560,19 @@ impl Runtime {
         node_id: NodeId,
         label: &Label,
     ) -> Result<(oak_abi::Handle, oak_abi::Handle), OakStatus> {
+        info!("creating Channel with label: {:?}", label);
+        if self.is_terminating() {
+            return Err(OakStatus::ErrTerminated);
+        }
+
+        // The label (and mere presence) of the newly created Channel is effectively public, so we
+        // must ensure that the label of the calling Node flows to both "public untrusted" and to
+        // the label of the Channel to be created.
+        self.validate_can_write_to_label(node_id, &Label::public_untrusted())?;
+        // We also additionally make sure that the label of the newly created Channel can be written
+        // to by the current Node, since in general this may be lower than "public untrusted".
```

Comment:
I don't understand this &ndash; what can be lower than public untrusted?  (Or does this anticipate future support for integrity labels?)

Reply:
The bottom of the information flow lattice is not public_untrusted, it is public_trusted.

Reply:
Yes, it anticipates support for integrity labels (in which case trusted is lower than untrusted, as @aferr points out), and also probably will be influenced by https://github.com/project-oak/oak/issues/1213 .